app/routes/ap-listing.js

```diff
@@ -0,0 +1,95 @@
+const { holdAdmin, schemeAdmin, dataView } = require('../auth/permissions')
+const api = require('../api')
+const convertToCSV = require('../convert-to-csv')
+const apListingSchema = require('../routes/schemas/ap-listing-schema')
+
+module.exports = [
+  {
+    method: 'GET',
+    path: '/report-list/ap-listing',
+    options: {
+      auth: { scope: [holdAdmin, schemeAdmin, dataView] },
+      handler: async (request, h) => {
+        return h.view('ap-listing-report')
```

Comment:
its a bit pedantic / ocd but personally I'd prefer the view URL to be the same as the path. just makes the code easier to understand how it all links together.

Reply:
to do this in views, add the 'ap-listing-report' file to a subfolder of 'views' called 'report-list' and rename it to 'ap-listing'